src/AutoRest.CSharp/Mgmt/Output/MgmtReferenceType.cs

```diff
@@ -14,23 +13,74 @@ namespace AutoRest.CSharp.Mgmt.Output
 {
     internal class MgmtReferenceType : MgmtObjectType
     {
+        private static HashSet<string> PropertyReferenceTypeModels = new HashSet<string>
```

Comment:
Can we put these into a config file?

Reply:
would prefer to keep them close to the actual logic